src/main/kotlin/org/rust/ide/intentions/AddImportForMatchArmPatternIntention.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.codeInsight.intention.PriorityAction
+import com.intellij.codeInsight.intention.PriorityAction.Priority
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.ide.inspections.import.AutoImportFix
+import org.rust.ide.intentions.AddImportForMatchArmPatternIntention.Context
+import org.rust.lang.core.psi.RsMatchArm
+import org.rust.lang.core.psi.RsPatBinding
+import org.rust.lang.core.psi.RsPatIdent
+import org.rust.lang.core.psi.ext.ancestorOrSelf
+import org.rust.lang.core.psi.ext.ancestorStrict
+import org.rust.lang.core.psi.ext.isAncestorOf
+
+class AddImportForMatchArmPatternIntention : RsElementBaseIntentionAction<Context>(), PriorityAction {
+
+    override fun getText(): String = "Import"
+    override fun getFamilyName(): String = "Add import for path in match arm"
```

Comment:
BTW why only match arms? There are other patterns as well where it can be useful
For example, `if let` construction

I don't force to do it in this PR but if you agree, let's change wording and user visible texts and `AddImportForMatchArmPatternIntention` class name